### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing appropriate information on changing the status of the checkboxes in "Create Authentication Files dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1523]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to checkboxes and check/uncheck .
6. observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA are not announcing appropriate information on changing the status of the checkboxes

**Jaws Behaviour:**
Table
Not Selected True
Column 1 ,Row 2
Not Selected True

**NVDA Behaviour**
1  checked  row 1  column 1
1  checked  row 2  column 1
 
1  pressed  checked  row 1  Subscription id  column 3
0
table
 
0  pressed  row 1  Subscription id  column 3
1
checked

**Expected Result:**
Jaws and NVDA should announce the status of the checkboxes ie checked/unchecked at the time of user action

Jaws and NVDA should announce as
Table
Not Selected checked 
Column 1 ,Row 1
Not Selected unchecked
Column 1 ,Row 2

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![create authntication files-check box](https://user-images.githubusercontent.com/22956130/36189502-eaef17f4-1178-11e8-9c59-6474d63d1676.jpg)
![create authntication files-check box-nvda](https://user-images.githubusercontent.com/22956130/36189505-eeb87498-1178-11e8-8583-2047c6e998a0.jpg)
![45bee2d4](https://user-images.githubusercontent.com/22956130/36189507-f25afc74-1178-11e8-8568-cd627c03f292.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by soumyatippapuram** [comment]
This issue is still reproducing in latest test pass 
(IntelliJ IDEA 2019.1.3  Build #IU-191.7479.19)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749972
